### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JannThomas**
This updates the README.md to reflect the new methods and also updates the cannot implement section with information about where it cannot be implemented.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,27 +49,33 @@ Introspection
...
-ScrollView | UIScrollView | NSScrollView | `.introspectScrollView()`
-NavigationView | UINavigationController | _N/A_ | `.introspectNavigationController()`
+NavigationView (StackNavigationViewStyle) | UINavigationController | _N/A_ | `.introspectNavigationController()`
+NavigationView (DoubleColumnNavigationViewStyle) | UISplitViewController | N/A | `.introspectSplitViewController()`
 _Any embedded view_ | UIViewController | _N/A_ | `.introspectViewController()`
-TabView | UITabBarController | _N/A_ | `.introspectTabBarController()`
+ScrollView | UIScrollView | NSScrollView | `.introspectScrollView()`
+List | UITableView | NSTableView | `.introspectTableView()`
+View in List | UITableViewCell | NSTableCellView | `introspectTableViewCell()`
+TabView | UITabBarController | NSTabView | `.introspectTabBarController()` (iOS) / `.introspectTabView` (macOS)
```
Comment (SplittyDev) [inline README.md:58]:
```suggestion
TabView | UITabBarController | NSTabView | `.introspectTabBarController()` (iOS)<br/>`.introspectTabView` (macOS)
```

File: README.md [hunk 2]
```diff
@@ -49,27 +49,33 @@ Introspection
 
 SwiftUI | UIKit | AppKit | Introspect
 --- | --- | --- | ---
-List | UITableView | NSTableView | `.introspectTableView()`
-ScrollView | UIScrollView | NSScrollView | `.introspectScrollView()`
-NavigationView | UINavigationController | _N/A_ | `.introspectNavigationController()`
+NavigationView (StackNavigationViewStyle) | UINavigationController | _N/A_ | `.introspectNavigationController()`
+NavigationView (DoubleColumnNavigationViewStyle) | UISplitViewController | N/A | `.introspectSplitViewController()`
```
Comment (SplittyDev) [inline README.md:53]: Please use the same styling for `N/A` everywhere.
Sometimes it's `_N/A_` and sometimes it's `N/A`.

**Review by SplittyDev — CHANGES_REQUESTED** [review]
A few 💅 suggestions, other than that LGTM

File: README.md (hunk 1 above)
Comment (JannThomas) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (JannThomas) [reply README.md]: Done

**Review by SplittyDev — APPROVED** [review]
LGTM 🚀

**Pull request merged**